agent/agent_endpoint.go

```diff
@@ -75,6 +77,17 @@ func (s *HTTPServer) AgentSelf(resp http.ResponseWriter, req *http.Request) (int
 	}, nil
 }
 
+// enablePrometheusOutput will look for Prometheus mime-type or format Query parameter the same way as Nomad
+func enablePrometheusOutput(req *http.Request) bool {
+	if contentType := req.Header.Get("Accept"); contentType == "text/plain; version=0.0.4; charset=utf-8" {
```

Comment:
This seems like a super generic ContentType to be switching on. The version part is not exactly common but this is not really an obvious signal that it's Prometheus making a request.

Given that Nomad doesn't support that, I'd prefer we keep this the same as Nomad for now and only switch on the explicit format.

Reply:
@banks Ok, I'll remove this in next patch, I added this from suggestion of @iksaif who is our local Prometheus expert, but we can live without that